src/components/Form/InputRaw.js

```diff
@@ -91,32 +97,49 @@ const InputRaw = (props) => {
     return null;
   };
 
+  const renderSpinner = () => {
+    if (!showSpinner) {
+      return null;
+    }
+
+    return (
+      <Spinner
+        brand
+        className="slds-input__spinner slds-m-right_xx-small"
+        size="x-small"
+      />
+    );
+  };
+
   const sldsClasses = [
-    { 'slds-input_bare': bare },
-    { 'slds-input': !bare },
+    { 'slds-has-input-focus': isFocused },
+    bare ? 'slds-input_bare' : 'slds-input',
```

Comment:
i suppose this should be `slds-input--bare`

Reply:
Nope, `.slds-input_bare` is correct, see https://lightningdesignsystem.com/components/input/

Reply:
okay, ignore my comment: react-lds is already on 2.3.1